lasagne/tests/layers/test_recurrent.py

```diff
@@ -1,12 +1,13 @@
 import pytest
+
 from lasagne.layers import RecurrentLayer, LSTMLayer, CustomRecurrentLayer
-from lasagne.layers import InputLayer, DenseLayer, GRULayer, Gate
+from lasagne.layers import InputLayer, DenseLayer, GRULayer, Gate, Layer
 from lasagne.layers import helper
 import theano
 import theano.tensor as T
 import numpy as np
 import lasagne
-
+from mock import Mock
```

Comment:
Oops.  The tests have lots of PEP-8/pyflakes warnings on my system, so I overlooked this in the midst of the others.  Fixed it.